File: LinkedList/src/Node.java

public class Node {
	private int value = 0;
	private Node next;
	
	public Node(){
		value = 0;
		next = null;
	}
	
	public int getValue(){
		return value;
	}
	public Node getNext(){
		return next;
	}
	public void setValue(int val){
		value = val;
	}
	public void setNext(Node nextNode){
		next = nextNode;
	}
}

File: Stack/src/Node.cpp
//
//  Node.cpp
//  Practice
//
//  Created by Ryan Wilcox on 7/29/12.
//  Copyright (c) 2012 Ryan Wilcox. All rights reserved.
//

#include "Node.h"

Node::Node()
    :listItem(), next(NULL)
{       }

Node::Node(int& item, Node* nextNode)
    :listItem(item),next(nextNode)
{       }

Node::~Node()
{       }

void Node::setListItem(int& item)
{ listItem = item; }

void Node::setNext(Node* nextNode)
{ next = nextNode; }

int Node::getListItem() const
{ return listItem; }

Node* Node::getNext() const
{ return next; }


